### PR TITLE
New salad recipe for practice purposes

### PR DESCRIPTION
**Pull request by chlotastrophie**

### CODE REVIEW

**Comment by noahsark769** [comment]
Ship it :ship:

File: salad.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+Dijon-style mustard
+Honey
+Ground pepper
+
+Instructions
+1. Rinse and tear lettuce; place into a bowl.
+2. Halve and core the pear, then put each half into slices.
+3. Divide the pear slices and pomegranate seeds among the two bowls and mix.
+4. Combine the vegetable oil, pomegranate juice, lemon juce, mustard, honey, and pepper into a saucepan.
+5. Bring mixture to a boil over high heat; reduce heart and simmer, stirring frequently until the dressing thickens slighly
```
Comment (noahsark769) [inline salad.md]: You did spell "slightly" wrong though haha

**Comment by chlotastrophie** [comment]
Yay! I remembered how to do it lol.

File: salad.md (hunk 1 above)
Comment (chlotastrophie) [inline salad.md]: NOOOOOOO.

**Pull request merged**